charts/kubefed/charts/controllermanager/templates/webhook.yaml

```
@@ -152,15 +152,15 @@ stringData:
   tls.key: {{ $cert.Key | quote }}
 {{- else }}
 ---
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: {{ .Values.certManager.apiGroup }}/{{ .Values.certManager.apiVersion }}
```

This would be better as a helper actually but the idea is to inspect the current version of the cert-manager API available, rather than relying on the user setting it through values.

```suggestion
{{- if (.Capabilities.APIVersions.Has "cert-manager.io/v1") }}
    apiVersion: cert-manager.io/v1
{{- else }}
    apiVersion: certmanager.k8s.io/v1alpha1
{{- end }}
```